### DISCUSSION TITLE
Navigation shortcut

### DISCUSSION THREAD

**Discussion by batrot-nat** [sumatrapdfreader/sumatrapdf#3870] [Q&A]
With the latest build, as well as with the pre-release, Ctrl+Page Down does not advance the current file to the next page. Instead, the combo takes me to the next tab.
Help.

**Comment by kjk** [comment]
You can use `n` and `p`.
I've re-assigned `Ctrl + PageUp / PageDown` to navigate tabs.
You can re-assign them back via customization as described in https://www.sumatrapdfreader.org/docs/Customizing-keyboard-shortcuts

```
Shortcuts [
	[
		Cmd = CmdGoToNextPage
		Key = Ctrl + PageDown
	]
	[
		Cmd = CmdGoToPrevPage
		Key = Ctrl + PageUp
	]
]
```
